From a34aae65caa5901b276dd9bf27f59fbb44250248 Mon Sep 17 00:00:00 2001
From: Adrian DC <radian.dc@gmail.com>
Date: Thu, 6 Sep 2018 22:07:56 +0200
Subject: [PATCH 13/14] Revert "libcameraservice: Disable external provider for
 legacy HAL1"

 * A device-side alternative was identified,
    held under the following Change-Id:
    I00c67de1b94a1f7eca1bf458d1d0e4de3507aa81

This reverts commit 1d2a8a543dc935dbac109daa1e46ab56d5033542.

Change-Id: Ib5e87abd54442cd94b60db9570acb6ebcab14650
---
 .../camera/libcameraservice/common/CameraProviderManager.cpp  | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/services/camera/libcameraservice/common/CameraProviderManager.cpp b/services/camera/libcameraservice/common/CameraProviderManager.cpp
index 196c2a6d1..0ce4318e7 100644
--- a/services/camera/libcameraservice/common/CameraProviderManager.cpp
+++ b/services/camera/libcameraservice/common/CameraProviderManager.cpp
@@ -37,9 +37,7 @@ namespace {
 // Hardcoded name for the passthrough HAL implementation, since it can't be discovered via the
 // service manager
 const std::string kLegacyProviderName("legacy/0");
-#ifndef NO_CAMERA_SERVER
 const std::string kExternalProviderName("external/0");
-#endif
 
 } // anonymous namespace
 
@@ -71,9 +69,7 @@ status_t CameraProviderManager::initialize(wp<CameraProviderManager::StatusListe
 
     // See if there's a passthrough HAL, but let's not complain if there's not
     addProviderLocked(kLegacyProviderName, /*expected*/ false);
-#ifndef NO_CAMERA_SERVER
     addProviderLocked(kExternalProviderName, /*expected*/ false);
-#endif
 
     return OK;
 }
-- 
2.17.1

